doc/contributing/coding-conventions.xml

```diff
@@ -652,6 +662,16 @@ args.stdenv.mkDerivation (args // {
          </variablelist>
         </listitem>
        </varlistentry>
+       <varlistentry>
+        <term>
+         If it’s a <emphasis>desktop theme</emphasis>:
```

Comment:
Desktop theme is ambiguous. Icon themes are also themes; so are [sound themes](https://specifications.freedesktop.org/sound-theme-spec/latest/). Maybe GTK and Qt themes, or whatever?

Reply:
It's not just GTK and Qt though. It could be E17 themes, a login manager theme, maybe even a Grub or Plymouth one.